packages/driver/cypress/integration/issues/4062_spec.js

```diff
@@ -0,0 +1,17 @@
+describe('4062', () => {
+  after(() => {
+    cy.get('h2').first()
+  })
+
+  it('make a test fail', () => {
+    const inputValue = 'fake'
+
+    cy.on('fail', (err) => {
+      expect(err.message).to.contain(inputValue)
+      expect(err.name).to.eql('AssertionError')
+    })
+
+    cy.visit('https://example.cypress.io/commands/actions')
```

Comment:
Visiting this site will add unnecessary time to our tests - waiting for the load event and potentially introduced flake if the services hosting the site are ever down. This test should fail without this visit command anyway, so could just be removed altogether.

```suggestion
```